build.sbt

```diff
@@ -3,55 +3,24 @@ import com.typesafe.tools.mima.core.{ProblemFilters, ReversedMissingMethodProble
 val scala3Version = "3.1.0"
 
 ThisBuild / organization := "org.typelevel"
+ThisBuild / tlBaseVersion := "3.0"
 ThisBuild / scalaVersion := scala3Version
 ThisBuild / crossScalaVersions := Seq(scala3Version)
-ThisBuild / mimaFailOnNoPrevious := false
 ThisBuild / updateOptions := updateOptions.value.withLatestSnapshots(false)
 
-val previousVersion = "3.0.0"
-
 // GHA configuration
-
-ThisBuild / githubWorkflowJavaVersions := List(JavaSpec.temurin("8"))
-ThisBuild / githubWorkflowArtifactUpload := false
+ThisBuild / tlCiReleaseBranches := Seq("main")
 ThisBuild / githubWorkflowBuildMatrixFailFast := Some(false)
```

Comment:
Do we still need this? It seems like it's the only setting left from `githubWorkflow`

Reply:
The default is `None` which is essentially `true`—fail fast is enabled (the first failing job cancels the rest). Up to you how you want to run your build :)

Reply:
Let's remove it 😄